model_building/wrapper_experiment_configuration.py

```diff
@@ -599,6 +615,7 @@ def _train(self):
         """
         if self._hyperopt_trained:  # do not run Hyperopt again for the same exp.conf.
             SFSExperimentConfiguration._train(self)
+            self._logger.info("Hyperopt already run")
```

Comment:
I'd put this in debug mode, not info, maybe with a slightly more detailed message

Reply:
Now in debug, message slightly changed